content/en/docs/install-setup/_index.html

```diff
@@ -14,239 +14,7 @@
 menu="Docs"
 weight="20"
 layout="home"
-
+url="/docs/install-setup/installing/"
```

Comment:
:+1, good one. Maybe we can extract this one as a dedicated pull request to fix the code duplication asap